coriolis/tests/conductor/rpc/data/migrate_instances_config.yml

```diff
@@ -0,0 +1,221 @@
+      depends_on: ['CREATE_INSTANCE_DISKS']
+    - type: REPLICATE_DISKS
+      depends_on: ['DEPLOY_MIGRATION_SOURCE_RESOURCES', 'DEPLOY_MIGRATION_TARGET_RESOURCES']
+    - type: REPLICATE_DISKS
+      depends_on: ['REPLICATE_DISKS']
+    - type: DELETE_MIGRATION_SOURCE_RESOURCES
+      depends_on: ['DEPLOY_MIGRATION_SOURCE_RESOURCES', 'REPLICATE_DISKS']
+      on_error: True
+    - type: CLEANUP_INSTANCE_SOURCE_STORAGE
+      depends_on: ['REPLICATE_DISKS', 'DELETE_MIGRATION_SOURCE_RESOURCES']
+      on_error: True
+    - type: DELETE_MIGRATION_TARGET_RESOURCES
+      depends_on: ['DEPLOY_MIGRATION_TARGET_RESOURCES', 'REPLICATE_DISKS']
+      on_error: True
+    - type: DEPLOY_INSTANCE_RESOURCES
+      depends_on: ['REPLICATE_DISKS', 'DELETE_MIGRATION_TARGET_RESOURCES']
+    - type: FINALIZE_INSTANCE_DEPLOYMENT
+      depends_on: ['DEPLOY_INSTANCE_RESOURCES']
+    - type: CLEANUP_FAILED_INSTANCE_DEPLOYMENT
+      depends_on: ['DEPLOY_INSTANCE_RESOURCES', 'FINALIZE_INSTANCE_DEPLOYMENT']
+      on_error_only: True
+    - type: CLEANUP_INSTANCE_TARGET_STORAGE
+      depends_on: ['CREATE_INSTANCE_DISKS', 'CLEANUP_INSTANCE_SOURCE_STORAGE', 'DELETE_MIGRATION_TARGET_RESOURCES', 'CLEANUP_FAILED_INSTANCE_DEPLOYMENT']
+      on_error_only: True
+
+- config:
+    shutdown_instances: True
+  expected_tasks:
+    - type: DEPLOY_MIGRATION_SOURCE_RESOURCES
+      depends_on: ['GET_INSTANCE_INFO', 'VALIDATE_MIGRATION_SOURCE_INPUTS']
+    - type: CREATE_INSTANCE_DISKS
+      depends_on: ['VALIDATE_MIGRATION_SOURCE_INPUTS', 'VALIDATE_MIGRATION_DESTINATION_INPUTS']
+    - type: DEPLOY_MIGRATION_TARGET_RESOURCES
+      depends_on: ['CREATE_INSTANCE_DISKS']
+    - type: REPLICATE_DISKS
+      depends_on: ['DEPLOY_MIGRATION_SOURCE_RESOURCES', 'DEPLOY_MIGRATION_TARGET_RESOURCES']
+    - type: SHUTDOWN_INSTANCE
+      depends_on: ['REPLICATE_DISKS']
+    - type: REPLICATE_DISKS
+      depends_on: ['SHUTDOWN_INSTANCE']
+    - type: DELETE_MIGRATION_SOURCE_RESOURCES
+      depends_on: ['DEPLOY_MIGRATION_SOURCE_RESOURCES', 'REPLICATE_DISKS']
+      on_error: True
+    - type: CLEANUP_INSTANCE_SOURCE_STORAGE
+      depends_on: ['REPLICATE_DISKS', 'DELETE_MIGRATION_SOURCE_RESOURCES']
+      on_error: True
+    - type: DELETE_MIGRATION_TARGET_RESOURCES
+      depends_on: ['DEPLOY_MIGRATION_TARGET_RESOURCES', 'REPLICATE_DISKS']
+      on_error: True
+    - type: DEPLOY_INSTANCE_RESOURCES
+      depends_on: ['REPLICATE_DISKS', 'DELETE_MIGRATION_TARGET_RESOURCES']
+    - type: FINALIZE_INSTANCE_DEPLOYMENT
+      depends_on: ['DEPLOY_INSTANCE_RESOURCES']
+    - type: CLEANUP_FAILED_INSTANCE_DEPLOYMENT
+      depends_on: ['DEPLOY_INSTANCE_RESOURCES', 'FINALIZE_INSTANCE_DEPLOYMENT']
+      on_error_only: True
+    - type: CLEANUP_INSTANCE_TARGET_STORAGE
+      depends_on: ['CREATE_INSTANCE_DISKS', 'CLEANUP_INSTANCE_SOURCE_STORAGE', 'DELETE_MIGRATION_TARGET_RESOURCES', 'CLEANUP_FAILED_INSTANCE_DEPLOYMENT']
+      on_error_only: True
+
+- config:
+    skip_os_morphing: False
```

Comment:
After adding that `has_os_morphing_pool` option, I'd prefer if we'd have the following extra cases:
- `skip_os_morphing: False`, `has_os_morphing_pool: False`, `has_origin/destination_pool: True`;
- `skip_os_morphing: False`, `has_os_morphing_pool: True`, `has_origin/destination_pool: True`;
- `skip_os_morphing: False`, `has_os_morphing_pool: True`, `has_origin/destination_pool: False`;
- `has_os_morphing_pool: True`  (implicitly, `skip_os_morphing` is `True` here)

Reply:
The new cases have been added.